webpack/components/PermissionDenied/index.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { Button, Icon } from 'patternfly-react';
+import PropTypes from 'prop-types';
+
+const PermissionDenied = ({ missingPermissions }) => {
```

Comment:
I hate to ask this but should this live in foreman?

Reply:
@waldenraines Idk.. what's our rule for what should live in Foreman vs. Katello?   This component is basically a copy of a Foreman ERB template, re-written in React.  To me it makes sense to have it live here, because that's where we are using the data.  If it lived in Foreman what would the difference be?  We would just import it from a different location?

If we're going to update Foreman anyway, it would also make sense to have the API give us an array of missing permissions, rather than parsing them from the error message string..

Reply:
I think we should both things mentioned above.  We don't have to port this to foreman right now but, to me, it makes sense that it lives there since it will eventually be used in both foreman and katello.

@amirfefer @sharvit thoughts?

Reply:
Maybe I'm beginning to realize what the `move_to_foreman` folder is for...